datahub-web-react/src/app/entityV2/shared/tabs/Documentation/DocumentationTab.tsx

```diff
@@ -137,15 +122,25 @@ export const DocumentationTab = ({ properties }: { properties?: Props }) => {
             ) : (
                 <EmptyTabWrapper>
                     <EmptyTab tab="documentation" hideImage={false}>
-                        {!hideLinksButton && <AddLinkModal buttonType="transparent" refetch={refetch} />}
-                        <PrimaryButton
-                            type="primary"
-                            size="large"
+                        {!hideLinksButton && <AddLinkModal refetch={refetch} />}
+                        <Button
                             data-testid="add-documentation"
                             onClick={() => routeToTab({ tabName: 'Documentation', tabParams: { editing: true } })}
                         >
                             <PlusOutlined /> Add Documentation
-                        </PrimaryButton>
+                        </Button>
+                        {shouldShowInferenceButton && (
+                            <InferDocsButton
+                                style={{ display: 'inline-block', top: 3, marginLeft: 8, height: 40 }}
+                                surface="entity-docs-tab"
+                                onClick={() =>
+                                    routeToTab({
+                                        tabName: 'Documentation',
+                                        tabParams: { editing: true, inferOnMount: true },
+                                    })
+                                }
+                            />
+                        )}
```

Comment:
i don't think we want this do we? also i don't think `shouldShowInferenceButton` exists in this component